pennylane_ls/FermionDevice.py

```diff
@@ -151,13 +154,14 @@ def check_job_status(self, job_id):
 
     def wait_till_done(self, job_id):
         while True:
-            time.sleep(2)
+            time.sleep(0.1)
```

Comment:
This is extremely short and I would prefer 1.

Reply:
Why is 0.1 very short? Shouldn't the server be able to respond on a milliseconds basis? Checking the status is a tiny http request.

Reply:
There is actually a whole chain of ifs, that is hidden in the code such that right now a sleep time of 0.1 seconds just does not make too much sense. Anyways, cold atoms will not work on a time scale that is much faster than 1s anyways so that this is not really a bottle neck right now...

Reply:
Ok, fine :-) Then feel free to change it to 1s before merging.